packages/server/lib/controllers/apiAuth.controller.ts

```
@@ -48,12 +49,13 @@ class ApiAuthController {
         };
 
         const activityLogId = await createActivityLog(log);
-        const logCtx = await logContextGetter.create(
-            { id: String(activityLogId), operation: { type: 'auth' }, message: 'Authorization API Key' },
-            { account: { id: accountId }, environment: { id: environmentId } }
-        );
 
+        let logCtx: LogContext | undefined;
         try {
+            logCtx = await logContextGetter.create(
```

if `create` fails no logs would be added but the error will be bubbling up. Not sure that is what we want since logging is very much internal

---

Right now it's safe, it's in the try/catch here, and currently in the client it's all try/catched too. Happy to discuss wether or not we should make it throw globally but right now should be good (unless I missed something) ☺️

---

I am not discussing the safety of it but the behavior if context creation fails. In this case no logs would be saved obviously but the context creation error will bubble up, which I am not sure it is what we want. Not a big deal though

---

ah yeah sorry I get it, so right now it's catched anyway in the `create()` function so no worries about that.
ultimately I wanted to remove it, but it's true that we could say it's not a critical piece and could leave it as a silent error